dynamodb:

fnmartinez opened the pull request:
```
+ Adding support for DynamoDB Tables
```

fnmartinez committed:
dynamodb:
	+ Adding support for DynamoDB Tables

etoccalino commented:
Awesome! DynamoDB integration looks great! I'll be holding on to this PR as we work on the integration tests.

fnmartinez committed:
master:
	* Adding parameters to constructor
	* Fixing names
	* Adding tests for the LiveTestDynamoDBTable
